src/main/java/org/folio/service/impl/TlrSettingsServiceImpl.java

```diff
@@ -39,6 +43,10 @@ public Optional<TlrSettings> updateTlrSettings(TlrSettings tlrSettings) {
       .findFirst()
       .map(entity -> tlrSettingsMapper.mapEntityToDto(
         tlrSettingsRepository.save(tlrSettingsMapper.mapDtoToEntity(
-          tlrSettings.id(entity.getId().toString())))));
+          tlrSettings.id(entity.getId().toString())))))
```

Comment:
Can we extract this code block into a method?
```
 tlrSettingsRepository.findAll(PageRequest.of(0, 1))
      .stream()
      .findFirst()
```

Reply:
extracted